server/app.py

```diff
@@ -17,12 +17,17 @@
 db = SQLAlchemy(app) 
 jwt = JWTManager(app)
 
-from resources import UserResources, GmailResources
+from models import UserModel, ProspectModel, TagModel, CampaignModel
+from resources import UserResources, GmailResources, CampaignResources
 
 api.add_resource(UserResources.UserRegister, '/register')
 api.add_resource(UserResources.UserLogin, '/login')
+api.add_resource(UserResources.UserCampaigns, '/campaigns')
 api.add_resource(GmailResources.GetAuthURL, '/gmail/get_auth_url')
 api.add_resource(GmailResources.Authorize, '/gmail/authorize')
 api.add_resource(GmailResources.GetGmailAddress, '/gmail/get_address')
+api.add_resource(CampaignResources.NewCampaign, '/campaign/new')
```

Comment:
Let's avoid roultes like this - a new campaign can be a POST in the /campaigns resource

Reply:
Can I just name it /campaign since it's only creating one at a time? /campaigns makes it sound like it's creating multiple

Reply:
The convention is to use plural (https://restfulapi.net/resource-naming/)